Extensions/show_picture_size.js

```diff
@@ -0,0 +1,71 @@
+//* TITLE Show picture size **//
```

Comment:
Should be "Show Picture Size" (title cased) for consistency with other extensions.